.gitignore

```diff
@@ -1,3 +1,6 @@
+# Skip __init__.py setting
+__init__.py
```

Comment:
`__init__.py` should not be in `.gitignore`, since it should generally not be ignored. Could you please remove this again? I believe this was added only after I reviewed the PR so I couldn't remark on it before.

Reply:
Adding `__init__.py` to `.gitignore` is indeed leading to problems, because `__init__.py` is missing from https://github.com/AstarVienna/METIS_Pipeline/pull/113 because it was ignored by git.